app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -226,4 +240,13 @@ void onPlayPause() {
         }
         controller.playPause();
     }
+
+    private boolean copyText(String text) {
+        ClipboardManager clipboardManager = ContextCompat.getSystemService(requireContext(), ClipboardManager.class);
+        if (clipboardManager != null) {
+            clipboardManager.setPrimaryClip(ClipData.newPlainText("AntennaPod", text));
+        }
+        Toast.makeText(requireContext(), getResources().getString(R.string.copied_to_clipboard), Toast.LENGTH_SHORT).show();
```

Comment:
Please use a Snackbar instead to be consistent with other parts of the app. We have a method called `((MainActivity) getActivity()).showSnackbarAbovePlayer` for that.